UI: Fixed Profiler metrics are not formatted correctly #7552

ShaileshParmar11 opened the pull request:
### Describe your changes :
<!-- Explain what you have done & tag your assigned issue !-->
I worked on the Fixing Profiler metrics that are not formatted correctly
Closes #7552
Closes [#7697](https://github.com/open-metadata/OpenMetadata/issues/7697)

#
### Type of change :
<!-- You should choose 1 option and delete options that aren't relevant -->
- [x] Bug fix
- [x] Improvement


#
### Frontend Preview (Screenshots) :
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/71748675/192280421-3a677825-f5ce-4de5-85d6-a4593e2cf64a.png">
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/71748675/192280479-600099f5-3445-4a94-922a-250065cd8988.png">
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/71748675/192280574-4961f58b-892f-440b-9114-e6fbf1ed2dfe.png">



#
### Checklist:
<!-- add an x in [] if done, don't mark items that you didn't do !-->
- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
<!-- Please see the contributing guidelines and then add your reviewer(s) !-->
<!--- OpenMetadata community thanks you for explaining your changes in detail !-->
<!--- If you are unsure of people to review your work, you can add anyone of these developers :) !-->
<!--- Frontend: @open-metadata/ui -->
<!--- Backend: @open-metadata/backend -->
<!--- Ingestion: @open-metadata/ingestion -->

TeddyCr commented:
@ShaileshParmar11 I think the issue with the decimal not being turned into percentage in the column drill down is not fixed. If you look at the column `CS_BILL_CUSTOMER_SK` you'll see that on the profiler landing page distinct = 40% but on the drill down distinct = 0.40%.

ShaileshParmar11 commented:
> @ShaileshParmar11 I think the issue with the decimal not being turned into percentage in the column drill down is not fixed. If you look at the column `CS_BILL_CUSTOMER_SK` you'll see that on the profiler landing page distinct = 40% but on the drill down distinct = 0.40%.

Hello I fixed the issue and also updated screenshort! Thanks
